test/unit/auditingTest.js

```diff
@@ -301,7 +301,7 @@ describe('Auditing', () => {
 
     after(async () => {
       config.auditing.auditEvents = JSON.parse(_restore)
-      await servers.map(s => s.close())
+      servers.map(s => s.close())
```

Comment:
We're still not waiting for the servers to close or handling any errors that may happen.